plugins/kubernetes-backend/src/service-locator/MultiTenantServiceLocator.ts

```diff
@@ -33,6 +34,7 @@ export class MultiTenantServiceLocator implements KubernetesServiceLocator {
   // As this implementation always returns all clusters serviceId is ignored here
   getClustersByEntity(
     _entity: Entity,
+    _requestContext: ServiceLocatorRequestContext,
```

Comment:
Try removing this line, does anything break?

Reply:
I'm kinda surprised by that, this must be a js/ts thing? I'm pretty sure this would have broken in Java/go

Reply:
I guess we can remove the breaking

Reply:
> this must be a js/ts thing? I'm pretty sure this would have broken in Java/go

Yep, totally!

Reply:
done!